hwangsae/org.hwangsaeul.hwangsae.gschema.xml

```diff
@@ -54,5 +54,10 @@
         access the recordings via HTTP.
       </description>
     </key>
+    <key name="recorder-id" type="s">
+      <default>""</default>
```

Comment:
Shouldn't it be `randomized-string` to generate id?

Reply:
According to source code it could be either empty or `randomized-string`, but let's use `randomized-string` as you suggested.